acceptance_tests/features/steps/print_file_steps.py

```diff
@@ -35,3 +47,62 @@ def _validate_print_file_content(sftp_utility, start_of_test, expected_csv_lines
         raise FileNotFoundError
 
     return True
+
+
+def _check_manifest_files_created(context):
+    with SftpUtility() as sftp_utility:
+        files = sftp_utility.get_all_files_after_time(context.test_start_local_datetime)
+
+        for _file in files:
+            if _file.filename.endswith(".csv"):
+                csv_file = _file
+                manifest_file = _get_matching_manifest_file(csv_file.filename, files)
+
+                if manifest_file is None:
+                    assert False, f'Failed to find manifest file for {csv_file.filename}'
+
+                actual_manifest = _get_actual_manifest(sftp_utility, manifest_file)
+                creation_datetime = actual_manifest['manifestCreated']
+                expected_manifest = _create_expected_manifest(sftp_utility, csv_file, creation_datetime)
+                tc.assertDictEqual(actual_manifest, expected_manifest)
+
+
+def _get_actual_manifest(sftp_utility, manifest_file):
+    actual_manifest_json = sftp_utility.get_file_contents_as_string(f'{Config.SFTP_DIR}/{manifest_file.filename}')
+    return json.loads(actual_manifest_json)
+
+
+def _get_matching_manifest_file(filename, files):
+    manifest_filename = filename.replace(".csv", ".manifest")
+
+    for _file in files:
+        if _file.filename == manifest_filename:
+            return _file
+
+    return None
+
+
+def _create_expected_manifest(sftp_utility, csv_file, created_datetime):
+    actual_file_contents = sftp_utility.get_file_contents_as_string(f'{Config.SFTP_DIR}/{csv_file.filename}')
+
+    md5_hash = hashlib.md5(actual_file_contents.encode('utf-8')).hexdigest()
+    expected_size = sftp_utility.get_file_size(f'{Config.SFTP_DIR}/{csv_file.filename}')
+
+    file = dict(
+        sizeBytes=expected_size,
+        md5sum=md5_hash,
+        relativePath='.\\',
+        name=csv_file.filename
+    )
+
+    manifest = dict(
+        schemaVersion=1,
+        files=[file],
```

Comment:
```suggestion
        files=[_file],
```

NB: an alternative to underscoring to avoid the reserved word `file` could be something like `fyle`, for example.